components/Voting/sessionPanel.styled.tsx

```diff
@@ -0,0 +1,84 @@
+/* tslint:disable:object-literal-sort-keys */
+/* tslint:disable:object-literal-key-quotes */
+import { breakpoint } from '../utils/styles/breakpoints'
+import { calcRem } from '../utils/styles/calcRem'
+import styled from '../utils/styles/theme'
+
+export const Panel = styled('li')(({ theme }) => ({
+  backgroundColor: theme.colors.lightGrey,
+  marginTop: '10px',
```

Comment:
There's a helper function that will let you use rems. Would be good if we are consistent. Check one of the other styled pages for its use but you do have it imported `calcRem`

`marginTop: calcRem(10),`